### DISCUSSION TITLE
Dumb Question - What is Roam Garden in relation to Roam Research?

### DISCUSSION THREAD

**Discussion by ruby-lorenze** [roam-garden/i#75] [Q&A]
As newbie to the Roam world I cant find a straightforward answer to what your Garden is and does!?!
1) Is it an extension / plugin to Roam version of new separate branch. ie. do I have be subscriber Roam Research ++ Roam.Garden.
2) What is Roam.Garden main feature that extends it beyond RR? Publication? Roam Research doesn't have that? Why would I want pay you to use it..how would it benefit me? ?
3) RR has mac osx App. I guess a progressive app? Will your garden features cascade in this app to? 
Thanks

**Comment by Stvad** [comment]
Hey @ruby-lorenze! 

To answer your questions:
1. It's a separate service that allows to publish content of your Roam Graph as a public static website. Roam Garden also has a Roam plugin that you can install to make publishing process easiert
2. Yes publication. People have many reason to publish their ideas on the internet 🤷‍♂️. Having a reference point for ideas, sharing information about products or documentation, selling a priviliged access to a set of information.
3. Roam Garden is an independent service, that said the publishing plugin will work inside desktop app
